mathics/builtin/natlang.py

```diff
@@ -1297,13 +1297,18 @@ class LanguageIdentify(Builtin):
         'pycountry',
     )
 
+    messages = {'langnotfound': 'Language code "``" not found.'}
     def apply(self, text, evaluation):
         'LanguageIdentify[text_String]'
         import langid  # see https://github.com/saffsd/langid.py
         # an alternative: https://github.com/Mimino666/langdetect
         import pycountry
         code, _ = langid.classify(text.get_string_value())
-        language = pycountry.languages.get(iso639_1_code=code)
+        try:
+            language = pycountry.languages.get(alpha_2=code)
+        except KeyError:
+            evaluation.message('LanguageIdentify', 'langnotfound', String(code))
+            return Symbol("$Failed")
```

Comment:
Nop... It was introduced to avoid an error in the travis tests, due to a former bug...